superbench/executor/executor.py

```diff
@@ -106,17 +110,47 @@ def __exec_benchmark(self, context, log_suffix):
                     logger.info('Executor succeeded in %s.', log_suffix)
                 else:
                     logger.error('Executor failed in %s.', log_suffix)
+                return benchmark.result
             else:
                 logger.error('Executor failed in %s, invalid context.', log_suffix)
-        except Exception:
+        except Exception as e:
+            logger.error(e)
             logger.error('Executor failed in %s.', log_suffix)
+        return None
+
+    def __get_benchmark_dir(self, benchmark_name, create=False):
+        """Get output directory for benchmark.
+
+        Args:
+            benchmark_name (str): Benchmark name.
+            create (bool): Create the directory or not.
+
+        Return:
+            str: Benchmark output directory.
+        """
+        benchmark_output_dir = Path(self._output_dir, 'benchmarks', benchmark_name)
```

Comment:
Each run will have different `self._output_dir`, right?

Reply:
yes, it's create here
https://github.com/microsoft/superbenchmark/blob/f1327bb788aeda7d755ee102e751a8b0edb038fa/superbench/common/utils/file_handler.py#L20